pdns/dnsbackend.hh

```diff
@@ -51,7 +52,7 @@ struct DomainInfo
   DNSName zone;
   time_t last_check;
   string account;
-  vector<string> masters;
+  vector<string> masters; // FIXME use ComboAddress
```

Comment:
I can contribute that cleanup if you want, and commit it to this PR?

Reply:
Would be awesome.